src/main/java/pages/RadioButton.java

```diff
@@ -0,0 +1,27 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+public class RadioButton {
+    private WebDriver driver;
+
+    public RadioButton (WebDriver driver){
+        this.driver = driver;
+    }
+    public void clickYes(){
+        Actions actions = new Actions(driver);
+        WebElement element = driver.findElement(By.id("yesRadio"));
+        actions.click(element).perform();
+    }
+    public void clickImpressive(){
+        Actions actions = new Actions(driver);
+        WebElement element = driver.findElement(By.id("impressiveRadio"));
+        actions.click(element).perform();
+    }
+    public String checkResult(){
```

Comment:
It is good to make three sections in page classes: Locators, Actions and Private methods. So in locators we just locate the elements, in actions we preform the actions on those elements, and in private methods we put things that we will use just in this class. Please refactor this class accordingly.

Reply:
And also Page Redirects section